tests/test_unclassified_job.py

```diff
@@ -26,3 +25,23 @@ def test_open_unclassified_failure_log(base_url, selenium):
     treeherder_page.open_next_unclassified_failure()
     logviewer_page = treeherder_page.job_details.open_logviewer()
     assert logviewer_page.is_job_status_visible
+
+
+@pytest.mark.nondestructive
+def test_view_unclassified_jobs(base_url, selenium):
+    page = TreeherderPage(selenium, base_url).open()
+    all_jobs = page.all_jobs
+
+    page.filter_unclassified_jobs()
+    filtered_jobs = page.all_jobs
+    jobstatus = page.select_random_job()
```

Comment:
This doesn't return a status but a job 'title'. Perhaps the property should be 'status' instead? Also, it doesn't select a job, it just picks a job at random and returns the title/status. This isn't a problem, but it shouldn't be in the page object - it should be in the test itself. See my other comment.